flarefly/fitter.py

```diff
@@ -736,6 +736,193 @@ def get_chi2_ndf(self):
         """
         return self.get_chi2()/self.get_ndf()
 
+    def get_raw_residuals(self, idx2=0):
+        """
+        Get the raw residuals (data_value - bkg_model_value) for all bins
+
+        Returns
+        -------------------------------------------------
+        residuals: array[float]
+            The residuals
+        """
+        residuals = []
+        chi2 = self.get_chi2_ndf()*self._ndf_
+        # access normalized data values and errors for all bins
+        binned_data = self._data_handler_.get_binned_data()
+        norm = self._data_handler_.get_norm()
+        data_values = binned_data.values()/norm
+        # access model predicted values for background
+        background_pdf_binned_ = zfit.pdf.BinnedFromUnbinnedPDF(self._background_pdf_[idx2],
+                                                                self._data_handler_.get_obs())
+        signal_fracs, _, _, _ = self.__get_all_fracs()
+        bkg_frac = 1 - signal_fracs[idx2]
```

Comment:
same here: this works only if you have a single background and a single signal pdf